sdk/python/kfp/dsl/tasks_group.py

```diff
@@ -454,20 +454,29 @@ def __init__(
             is_root=False,
         )
 
-        if isinstance(items, pipeline_channel.PipelineChannel):
-            self.loop_argument = for_loop.LoopArgument.from_pipeline_channel(
+        if isinstance(items, pipeline_channel.PipelineParameterChannel):
+            self.channel_type = str(items.channel_type)
```

Comment:
Do we need to set the `.channel_type` attribute? If I comment this out, the compilation test still passes.

Reply:
Missed that, done.